crates/swc_ecma_codegen/src/lib.rs

```diff
@@ -3480,7 +3480,7 @@ fn get_quoted_utf16(v: &str, ascii_only: bool, target: EsVersion) -> String {
                 let next = iter.peek();
 
                 match next {
-                    Some('1'..='9') => buf.push_str("\\x00"),
+                    Some('0'..='7') => buf.push_str("\\x00"),
```

Comment:
Sorry, no, it still fails!
```js
() => { "use strict"; 1; "\09"; }
```

Reply:
Yup: https://tc39.es/ecma262/multipage/ecmascript-language-lexical-grammar.html#prod-EscapeSequence

Sorry, it's gotta be `0..=9` to properly pass. `"\09"` is a [LegacyOctalEscapeSequence](https://tc39.es/ecma262/multipage/ecmascript-language-lexical-grammar.html#prod-LegacyOctalEscapeSequence), which is only valid in loose mode.